site/_resources/upgrading.md

```diff
@@ -71,6 +71,12 @@ $ kubectl apply -f examples/contour/02-job-certgen.yaml
 
 If your version of Contour is older than v1.13, please upgrade to v1.13 first, then upgrade to v1.14.0.
 
+This release includes an update to the Envoy service ports. Upgrade your Envoy service with the following:
+
+```bash
+$ kubectl apply -f examples/contour/02-service-envoy.yaml
+```
+
```

Comment:
I guess unfortunately there's not a great way to do this without a little bit of downtime. I'm not sure if/how we should communicate that. We don't make any guarantees right now about no-downtime upgrades so probably fine as-is.

Reply:
Yeah, this at least fixes the opened issue with some instructions so things aren't broken indefinitely, but there would still be some downtime

Reply:
Based on [line 41](https://github.com/projectcontour/contour/pull/3543/files#diff-a439f57ffcafffc9e8a4507a4fbb29e95d61abe4e2832b633f09994eaca3462fR41), where the easy way suggests there'll be downtime, the less easy way (which this change addresses) is still suggestive that there'll be no downtime.

Reply:
Would it make sense to pull this into a separate issue? Seems like we need some thinking around setting expectations for these upgrade instructions, as ultimately these are "example" manifests and not actually hardened upgrade instructions

Reply:
xref #3223, yeah I'd say let's get this omission merged and revisit upgrades more broadly.